source/commands.c

```diff
@@ -211,6 +211,74 @@ static inline void setColorMono(const message_t *msg) {
   setLedMono(ledColors, msg);
 }
 
+static inline void handleStickyEnabled(void) {
+  stickyKeysExist = 1;
+  if (!matrixEnabled) {
+    matrixEnable();
+    sendStatus();
+    // we changed shouldDisable by calling matrixEnable
+    // we must set it to 1 as the user wants backlight
+    // to be off
+    shouldDisable = 1;
```

Comment:
I'd suggest name this something more descriptive, one suggestion could be "backlightDisable". I know the code style / naming isn't super consistent in this repo, but I'd like to keep variable names as descriptive as possible

Reply:
I've renamed the variable to `backlightDisabled` I think it's slightly confusing in this if statement where `matrixEnabled` also makes an appearance - `if (!stickyKeysExist && backlightDisabled && matrixEnabled)` but I couldn't think of anything better, variable names aren't my strong suit :D

Reply:
haha me neither. I figured that this is at least a bit better than "shouldDisable" lol.

Reply:
yeah, it's definitely better, I just didn't realize how vague "shouldDisable" sounds while making the changes :D